<launch>
      <!-- -->
      <!-- detector -->
      <node name="detection_node" pkg="nodelet" type="nodelet" args="manager" output="screen"/>
      <!-- -->
      <!-- moving object detection -->
      <node name="detector_loader" pkg="nodelet" type="nodelet" 
            args="load moving_object_detector/MovingObjectDetector detection_node" output="screen">
            <remap from="image_raw" to="image_raw"/>
            <remap from="image_out" to="object_image"/>
            <remap from="objects_out" to="object_data"/>
            <param name="type" value="MOG"/>
            <param name="republish_image" value="true"/>
      </node>
      <!-- -->
      <!-- object drawing -->
      <node name="object_drawer_loader" pkg="nodelet" type="nodelet" 
            args="load object_drawer/ObjectDrawer detection_node" output="screen">
            <remap from="image_raw" to="object_image"/>
            <remap from="objects_in" to="object_data"/>
            <remap from="image_out" to="annotated_image"/>
             </node>
      <!-- -->
      <!-- viewer -->
      <node name="object_viewer" pkg="nodelet" type="nodelet" 
            args="standalone image_view/image" output="screen">
            <remap from="image" to="annotated_image"/>
      </node>
</launch>
